js/src/elements/segmented-controls/segmented-controls-element-define.ts

```diff
@@ -0,0 +1,42 @@
+import { TabContainerChangeEvent, TabContainerElement } from './segmented-controls-element';
+

```

Comment:
Rename the folder and files to not contain segmented control. We should keep the tab container nomenclature. We don't want to tie up the web component functionality with styling.

Reply:
✅